app/src/main/java/com/devshish/internship/presentation/ui/service/server/MediaSessionCallback.kt

```diff
@@ -0,0 +1,47 @@
+package com.devshish.internship.presentation.ui.service.server
+
+import android.net.Uri
+import android.os.Bundle
+import android.support.v4.media.session.MediaSessionCompat
+import android.support.v4.media.session.PlaybackStateCompat
+import timber.log.Timber
+
+class MediaSessionCallback(
+    private val mediaSession: MediaSessionCompat,
+    private val stateBuilder: PlaybackStateCompat.Builder
+) : MediaSessionCompat.Callback() {
+
+    private fun setState(
```

Comment:
You can make extension function like this:
![image](https://user-images.githubusercontent.com/23121846/137929790-8bf30922-cccf-4bdc-a125-fdffa9f5a125.png)